FusionIIIT/applications/globals/models.py

```diff
@@ -154,6 +161,11 @@ class ExtraInfo(models.Model):
         null=True, blank=True, upload_to='globals/profile_pictures')
     about_me = models.TextField(default='NA', max_length=1000, blank=True)
     date_modified = models.DateTimeField('date_updated', blank=True, null=True)
+    u_type = models.CharField(max_length=20, choices=Constants.USER_TYPES)
```

Comment:
changes this to user_type

Reply:
user_type is already used.

Reply:
Why do you need it then? You have just defined a duplicate column.

Reply:
Both user_type and u_type are different columns, user_type contains the designation of the user and the u_type includes the programme of the user. We can not use user_type instead of u_type as it serves its own purpose.